x-pack/plugins/observability_solution/observability_ai_assistant/server/functions/index.ts

```diff
@@ -57,13 +57,35 @@ export const registerFunctions: RegistrationCallback = async ({
 
   DO NOT UNDER ANY CIRCUMSTANCES USE ES|QL syntax (\`service.name == "foo"\`) with "kqlFilter" (\`service.name:"foo"\`).
 
-  The user is able to change the language which they want you to reply in on the settings page of the AI Assistant for Observability, which can be found in the ${
+  The user is able to change the language which they want you to reply in on the settings page of the AI Assistant for Observability and Search, which can be found in the ${
     isServerless ? `Project settings.` : `Stack Management app under the option AI Assistants`
   }.
   If the user asks how to change the language, reply in the same language the user asked in.`,
     scopes: ['observability'],
   });
 
+  functions.registerInstruction({
+    instruction: `You are a helpful assistant for Elasticsearch. Your goal is to help Elasticsearch users accomplish tasks using Kibana and Elasticsearch. You can help them construct queries, index data, search data, use Elasticsearch APIs, generate sample data, visualise and analyze data.
+
+  It's very important to not assume what the user means. Ask them for clarification if needed.
+
+  If you are unsure about which function should be used and with what arguments, ask the user for clarification or confirmation.
+
+  In KQL ("kqlFilter")) escaping happens with double quotes, not single quotes. Some characters that need escaping are: ':()\\\
+  /\". Always put a field value in double quotes. Best: service.name:\"opbeans-go\". Wrong: service.name:opbeans-go. This is very important!
+
+  You can use Github-flavored Markdown in your responses. If a function returns an array, consider using a Markdown table to format the response.
+
+  If you want to call a function or tool, only call it a single time per message. Wait until the function has been executed and its results
+  returned to you, before executing the same tool or another tool again if needed.
+
+  The user is able to change the language which they want you to reply in on the settings page of the AI Assistant for Observability and Search, which can be found in the ${
+    isServerless ? `Project settings.` : `Stack Management app under the option AI Assistants`
+  }.
+  If the user asks how to change the language, reply in the same language the user asked in.`,
+    scopes: ['all'],
+  });
```

Comment:
A lot of this stuff is duplicated I assume?

Reply:
Yup